go.mod

```diff
@@ -66,3 +65,5 @@ require (
 )
 
 replace golang.org/x/text v0.3.2 => golang.org/x/text v0.3.3
+
+replace github.com/longhorn/longhorn-engine v1.5.4 => github.com/ChanYiLin/longhorn-engine v1.4.0-rc1.0.20240311091244-3365ed4916a7
```

Comment:
This is not updated yet?

Reply:
Sure I can change it to Draft
I use to mention what this PR is depended on in the comment above

> Dependency PR:
longhorn-engine: https://github.com/longhorn/longhorn-engine/pull/1050